lang/luasocket5.3/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2009-2013 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luasocket
+PKG_SOURCE_VERSION:=733af884f1aa18ff469bf3c4d18810e815853211
+PKG_VERSION:=3.0-rc1-20190421
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_MIRROR_HASH:=915b06219cb79802fe547c1b1896bd7c5465b717549d6a0f38e2c0b4f3e7ae3e
+PKG_SOURCE_URL:=https://github.com/diegonehab/luasocket.git
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/luasocket5.3
+  SUBMENU:=Lua
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=LuaSocket 5.3
+  URL:=http://luasocket.luaforge.net/
+  MAINTAINER:=W. Michael Petullo <mike@flyn.org>
```

Comment:
Similar to the linked package, remove and add PKG_MAINTAINER above as well as license information.

Reply:
@neheb - I don't understand the request "remove and add PKG_MAINTAINER".  I'm confused when you say to remove it and add it at the same time.  I'll add the license.

Reply:
Check https://github.com/openwrt/packages/blob/master/utils/usbmuxd/Makefile

There's PKG_MAINTAINER and PKG_LICENSE at the top, not down here.

Reply:
I've moved the PKG_MAINTAINER and added PKG_LICENSE

Reply:
Place PKG_MAINTAINER above PKG_LICENSE

Reply:
While PKG_MAINTAINER works, the documented way is still MAINTAINER.  Please don't demand people change them "just because"  See https://openwrt.org/docs/guide-developer/packages

Reply:
@neheb - Per the "conversation" above, I was informed PKG_MAINTAINER can be blank.  I've done that, thus I think putting it above PKG_LICENSE is now no longer valid.